radicle-httpd/src/api/v1/projects.rs

```diff
@@ -360,14 +362,49 @@ async fn issue_handler(
         "id": issue_id,
         "author": issue.author(),
         "title": issue.title(),
-        "status": issue.status(),
-        "discussion": issue.comments().collect::<Vec<_>>(),
+        "state": issue.status(),
```

Comment:
We should probably call this status to match the method name and underlying type?

Reply:
`state` is what is expected, it'd end up like:
```
"state": { "status": "open|close", "reason": "" }
```

Reply:
Oh I see, then I should rename the method and type.

Reply:
here's the [type](https://github.com/radicle-dev/radicle-interface/blob/master/src/issue.ts#L21-L28) on interface for reference.